string traj_folder
string test_type
string curr_file